src/Configuration.ts

```diff
@@ -102,7 +102,7 @@ const defaultConfiguration: Configuration = {
   anchorIntervalInSeconds: 30,
 
   healthIntervalInSeconds: 30,
-  lowWalletBalanceBTC: 500,
+  lowWalletBalanceInBitcoin: 500,
```

Comment:
Where did this number com from? 
500 * ~6500USD/BTC = ~3.2 million USD

Reply:
Sorry when I did the number I was thinking it was going to be in dollars and didn't change it after I realized it was BTC. I'll lower this.